gobblin-restli/gobblin-throttling-service/gobblin-throttling-service-api/src/main/pegasus/org/apache/gobblin/restli/throttling/PermitRequest.pdsc

```diff
@@ -6,7 +6,8 @@
   "fields": [
     {"name": "resource", "type": "string", "doc": "Resource for which permits are needed."},
     {"name": "permits", "type": "long", "doc": "Number of permits needed."},
-    {"name": "minPermits", "type": "long", "doc": "Minimum number of useful permits.", "optional" : true},
-    {"name": "requestorIdentifier", "type": "string", "doc" : "Identifier of the service requesting the permits."}
+    {"name": "minPermits", "type": "long", "doc": "Minimum number of useful permits.", "optional" : true, "default": 0},
+    {"name": "requestorIdentifier", "type": "string", "doc" : "Identifier of the service requesting the permits."},
+    {"name": "version", "type": "int", "doc": "Request version.", "optional": true, "default": 0}
```

Comment:
Can we add more description on why this version is needed since ppl jumped in may not have enough context and it looks confused by just saying "request version"

Reply:
Fixed